assets/js/modules/tagmanager/datastore/__factories__/builders.js

```diff
@@ -146,7 +147,7 @@ export const defaultTagWeb = ( { accountId, containerId } = {} ) => ( {
 	blockingRuleId: null,
 	blockingTriggerId: null,
 	containerId,
-	fingerprint: Date.now().toString(),
```

Comment:
Thinking about this again, I don't think it really needs to be changed because there's no risk of a collision the way there was with faker. It doesn't really matter because I don't think we use this field at all, but it should be fine to use `Date` as it was before.

Reply:
We use snapshot tests to ensure these factories are accurate, so that's why I updated it.

I can select specific fields to test instead if you'd rather keep the `Date.now()`, but I'm always a bit wary to leave calls to `Date.now()` like that in tests 😅

Reply:
I updated the snapshot test to ignore the `fingerprint` key as it's not an important part of the test anyhow 👍🏻